Jenkinsfile

```diff
@@ -4,7 +4,7 @@ library "jenkinsLibrary@develop"
 buildPublishPipeline(
     slackChannel: "sandbox-github",
     asProjectRepo: "hammers",
-    asProjectVersion: "main", 
+    asProjectVersion: "feature/simulationStatus", 
```

Comment:
This should not be merged. I also question why this was changed?

Reply:
This changed will not be merged.  Currently, the only way I know to test a package before it's merged is to change the Jenkins file and push a tag (usually named Test-Tag) to the repo to trigger a build.  Is there a better way to do this without having to make this change?